doc/source/tune/tutorials/tune-output.rst

```diff
@@ -177,25 +177,25 @@ preferred way to interact with logging that happens in trainables. Leave you com
 
 .. _trainable-logging:
 
-How to configure logging of Tune Trainables?
---------------------------------------------
+How do you log arbitrary files from a Tune Trainable?
+-----------------------------------------------------
 
-By default, Tune only logs the *training result dictionaries* from your Trainable.
-However, you may want to visualize the model weights, model graph,
+By default, Tune only logs the *training result dictionaries* and *checkpoints* from your Trainable.
+However, you may want to save a file that visualizes the model weights/model graph,
```

Comment:
```suggestion
However, you may want to save a file that visualizes the model weights or model graph,
```

Reply:
I hope that doesn't change the meaning of what you want to say. In general, we try to avoid using slashes because they are ambiguous.